lib/validate.lib.ts

```diff
@@ -3,7 +3,7 @@
  * @param id the ID being validated
  * @returns {boolean} true if the given ID is a valid document ID, otherwise false
  */
-exports.isValidDocumentID = function(id) {
+export function isValidDocumentID(id) {
     const regExpDocumentID = new RegExp('^[a-fA-F0-9]{24}$');
     return id && typeof id === 'string' && id.length === 24 && regExpDocumentID.test(id);
 }
```

Comment:
goodness... we can probably kill some of this validate stuff w/ types when we refactor

Reply:
absolutely